storageapi/src/vespa/storageapi/messageapi/storagereply.cpp

```diff
@@ -14,7 +14,12 @@ StorageReply::StorageReply(const StorageCommand& cmd, ReturnCode code)
     if (cmd.getAddress()) {
         setAddress(*cmd.getAddress());
     }
-    setTrace(cmd.getTrace());
+    // TODD do we really need copy construction
```

Comment:
The `const`-ness of the `StorageCommand` argument is already a white lie, since we destructively (and silently) acquire its transport context by move. We could probably do the same thing with traces, though we should first ensure that there aren't any callers that are unaware of this implicit copy and that try to copy/move the command's trace into the reply explicitly after constructing it.